macos/FluentUI/Button/Button.swift

```diff
@@ -5,158 +5,254 @@
-	case primaryOutline	// Clear fill color, solid outline
-	case borderless		// Clear fill color, clear outline
+	/// No defined colors; user to specify.
+	case none
+
+	/// Accent color fill, white text/image.
+	case primary
+
+	/// Light mode: white fill, black text/image, light gray outline.
+	/// Dark mode: dark gray fill, white text/image, subtle gray outline.
+	/// Pressed: accent color fill, white text/image, no outline (same as primary style).
+	case secondary
+
+	/// Light mode: light gray fill, black text/image.
+	/// Dark mode: dark gray fill, white text/image.
+	case acrylic
+
+	/// Accent color text/image, no fill or outline.
+	case borderless
+}
+
+/// Combination of all formatting parameters.
+public struct ButtonFormat {
+	var size: ButtonSize
+	var style: ButtonStyle
+	var accentColor: NSColor?
+
+	public init(
+		size: ButtonSize = .large,
+		style: ButtonStyle = .primary,
+		accentColor: NSColor? = Colors.primary
+	)
+	{
+		self.size = size
+		self.style = style
+		self.accentColor = accentColor
+	}
+}
+
+// MARK: - Semantic Colors
+
+@objc(MSFButonColor)
+class ButtonColor: NSObject {
+	public static let brandForegroundDisabled = NSColor(named: "ButtonColors/brandForegroundDisabled", bundle: FluentUIResources.resourceBundle)
+	public static let brandBackgroundDisabled = NSColor(named: "ButtonColors/brandBackgroundDisabled", bundle: FluentUIResources.resourceBundle)
+	public static let neutralInverted = NSColor(named: "ButtonColors/neutralInverted", bundle: FluentUIResources.resourceBundle)
+	public static let neutralForeground2 = NSColor(named: "ButtonColors/neutralForeground2", bundle: FluentUIResources.resourceBundle)
+	public static let neutralBackground2 = NSColor(named: "ButtonColors/neutralBackground2", bundle: FluentUIResources.resourceBundle)
+	public static let neutralStroke2 = NSColor(named: "ButtonColors/neutralStroke2", bundle: FluentUIResources.resourceBundle)
+	public static let neutralForeground3 = NSColor(named: "ButtonColors/neutralForeground3", bundle: FluentUIResources.resourceBundle)
+	public static let neutralBackground3 = NSColor(named: "ButtonColors/neutralBackground3", bundle: FluentUIResources.resourceBundle)
 }
 
 // MARK: - Button
 
 /// A fluent styled button, with hover effects and a corner radius.
+@available(OSX 10.14, *)
 @objc(MSFButton)
 open class Button: NSButton {
-	/// Initializes a Fluent UI Button with a title, automatically colored to suit the style
+	/// Initializes a Fluent UI Button with a title only
 	/// - Parameters:
 	///   - title: String displayed in the button
-	///   - style: The ButtonStyle, defaulting to primaryFilled
-	@objc public init(title: String, style: ButtonStyle = .primaryFilled) {
-		super.init(frame: .zero)
-		initialize(title: title, image: nil, style: style)
+	///   - size: The ButtonSize, default .large
+	///   - style: The ButtonStyle, default .primary
+	///   - accentColor: The accent or highlight NSColor, defaulting to the app primary color
```

Comment:
The comment "accent or highlight" might be a bit confusing since these are separate concepts when it comes to NSColor

Reply:
The word was there to avoid an unhelpful circular statement.  Removed "highlight" to avoid confusion.